.gitignore

```diff
@@ -1,3 +1,4 @@
+backend/dist/*
 dist/*
 backend/dist/*
```

Comment:
We should probably only have this once

Reply:
whoops must have missed that when merging mah bad.